### PR TITLE
Rename certificates-refresh-interval to certificates-refresh-interval--in-ms in keyvault jca

### PR DESCRIPTION
**Pull request by zhichengliu12581**
Related issue: https://github.com/azure/azure-sdk-for-java/issues/23588 
```GitHub README, also found here.```
point 4

### CODE REVIEW

**Comment by zhichengliu12581** [comment]
@chenrujun Can you please help to review this pr and if we need to change to properties name.

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java [hunk 1]
```diff
@@ -122,7 +122,7 @@ public KeyVaultKeyStore() {
         String clientId = System.getProperty("azure.keyvault.client-id");
         String clientSecret = System.getProperty("azure.keyvault.client-secret");
         String managedIdentity = System.getProperty("azure.keyvault.managed-identity");
-        long refreshInterval = Optional.of("azure.keyvault.jca.certificates-refresh-interval")
+        long refreshInterval = Optional.of("azure.keyvault.jca.certificates-refresh-interval-in-ms")
```
Comment (chenrujun) [inline sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java:125]: How about not introduce breaking change by doing like this:
```
Stream.of("azure.keyvault.jca.certificates-refresh-interval-in-ms", "azure.keyvault.jca.certificates-refresh-interval")
.map(System::getProperty)
.map(Long::valueOf)
.filter(Object::nonNull)
.findFirst()
.orElse(0L)
```

And write unit test for this logic.

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**